src/main/java/domain/Bet/BetAmount.java

```diff
@@ -0,0 +1,45 @@
+package domain.Bet;
+
+import java.util.Objects;
+
+public final class BetAmount {
+
+    public static final int MIN_BET_AMOUNT = 500;
+    private final double value;
+
+    private BetAmount(final double value) {
+        this.value = value;
+    }
+
+    public static BetAmount from(final double value) {
+        if (value <= MIN_BET_AMOUNT) {
+            throw new IllegalArgumentException("베팅 금액은 최소 500원입니다.");
+        }
+        return new BetAmount(value);
+    }
```

Comment:
최소 금액이 500원이면 500원도 포함되는 것 처럼 느껴집니다.
조건을 미만으로 바꾸면 어떨까요?
```suggestion
    public static BetAmount from(final double value) {
        if (value < MIN_BET_AMOUNT) {
            throw new IllegalArgumentException("베팅 금액은 최소 500원입니다.");
        }
        return new BetAmount(value);
    }
```

Reply:
넵 괜찮은 것 같습니다. 변경하겠습니다 🙂